Library/Formula/xsd.rb

```diff
@@ -0,0 +1,49 @@
+class Xsd < Formula
+  desc "XML Data Binding for C++"
+  homepage "http://www.codesynthesis.com/products/xsd/"
+  url "http://www.codesynthesis.com/download/xsd/4.0/macosx/i686/xsd-4.0.0-i686-macosx.tar.bz2"
```

Comment:
Ok, I have fixed the parentheses, but I have not been able to get this to compile from source yet. Can we push this to the appropriate place and when/if I manage to compile it I will post a new formula to compile from source?